app/src/main/java/com/nervousfish/nervousfish/activities/MainActivity.java

```diff
@@ -195,28 +154,30 @@ public void onPairingButtonClicked(final View view) {
         final Intent intent;
         switch (view.getId()) {
             case R.id.pairing_menu_bluetooth:
-                intent = new Intent(this, ActivateBluetoothActivity.class);
+                this.enableBluetooth(true);
```

Comment:
I think it will be better to check and instantly go to the Bluetooth pairing screen within this method as opposed to the `enableBluetooth()` method